src/containers/Detail.js

```diff
@@ -0,0 +1,100 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView } from 'react-native';
+
+export const Detail = () => {
+  return (
+    <View style={styles.container}>
```

Comment:
- Not find styles container.
- If style is not significantly changing then just use ScrollView instead.